Adapters/Vungle/ISVungleAdapter/ISVungleAdapter.m

```diff
@@ -102,74 +79,39 @@ - (NSString *)sdkName {
 
 - (instancetype)initAdapter:(NSString *)name {
     self = [super initAdapter:name];
-    
+
     if (self) {
         if (initCallbackDelegates == nil) {
             initCallbackDelegates = [ConcurrentMutableSet<ISNetworkInitCallbackProtocol> set];
         }
-        
-        // Rewarded video
-        _rewardedVideoPlacementIdToSmashDelegate        = [ConcurrentMutableDictionary dictionary];
-        _rewardedVideoPlacementIdToServerData           = [ConcurrentMutableDictionary dictionary];
-        _rewardedVideoServerDataToSmashDelegate         = [ConcurrentMutableDictionary dictionary];
-        _rewardedVideoPlacementIdsForInitCallbacks      = [ConcurrentMutableSet set];
-        
-        // Interstitial
-        _interstitialPlacementIdToSmashDelegate         = [ConcurrentMutableDictionary dictionary];
-        _interstitialPlacementIdToServerData            = [ConcurrentMutableDictionary dictionary];
-        _interstitialServerDataToSmashDelegate          = [ConcurrentMutableDictionary dictionary];
-        
-        // Banner
-        _bannerPlacementIdToSmashDelegate               = [ConcurrentMutableDictionary dictionary];
-        _bannerPlacementIdToSize                        = [ConcurrentMutableDictionary dictionary];
-        _bannerPlacementIdToViewController              = [ConcurrentMutableDictionary dictionary];
-        _bannerPlacementIdToBannerState                 = [ConcurrentMutableDictionary dictionary];
-        _bannerPlacementIdToServerData                  = [ConcurrentMutableDictionary dictionary];
-        _bannerServerDataToSmashDelegate                = [ConcurrentMutableDictionary dictionary];
     }
-    
+
     return self;
 }
 
 - (void)initSDKWithAppId:(NSString *)appId {
-    
     // add self to the init delegates only in case the initialization has not finished yet
     if ((_initState == INIT_STATE_NONE) || (_initState == INIT_STATE_IN_PROGRESS)) {
```

Comment:
This should only check the State NONE.
and if the state is in progress we should early return;

Only if My other comment is Valid https://github.com/Vungle/levelplay-ios-adapters/pull/1/files#r991537655